package helgestenstrom;

import org.junit.Test;

import static org.junit.Assert.fail;

public class mainTests {
    @Test
    public void summation() throws Exception {
        fail();
    }
}
